db/init.sql

```diff
@@ -0,0 +1,150 @@
+DROP TABLE IF EXISTS "competition" CASCADE;
+DROP TABLE IF EXISTS "game" CASCADE;
+DROP TABLE IF EXISTS "invitation" CASCADE;
+DROP TABLE IF EXISTS "registration" CASCADE;
+DROP TABLE IF EXISTS "team" CASCADE;
+DROP TABLE IF EXISTS "user" CASCADE;
+
+CREATE TYPE shirt_size_enum AS ENUM (
+    'S', 'M', 'L', 'XL', 'XXL'
+);
+
+CREATE TYPE pizza_choice_enum AS ENUM (
+    'Cheese', 'Pepperoni', 'Bacon'
+);
+
+CREATE TYPE client_language_enum AS ENUM (
+    'cpp', 'js', 'lua', 'py', 'cs', 'java'
+);
+
+CREATE TYPE invitation_response_enum AS ENUM (
+    'accepted', 'declined'
+);
+
+CREATE TABLE competition
+(
+  id serial NOT NULL,
+  name character varying(64) NOT NULL,
+  cost integer,
+  description character varying(64),
```

Comment:
I'd add a default here something like `Default competition description`, that way its obvious if a web page shows the default hasnt been changed.